website/docs/advanced-concepts/granular-access-control/reference/permissions.mdx

```diff
@@ -0,0 +1,717 @@
+---
+description: Permissions in Granular Access Control in Appsmith
+title: Permissions
```

Comment:
On this whole page, there's no such thing as Application datasource, all datasources are workspace wide.

Reply:
Naming typo.